noxfile.py

```diff
@@ -0,0 +1,166 @@
+"""
+Flexible test automation with Python for cachi2.
+
+To run all sessions, run the following command:
+$ nox
+
+To run a specific session, run the following command:
+$ nox -s <session-name>
+
+To run a session with additional arguments, run the following command:
+$ nox -s <session-name> -- <additional-arguments>
+
+To list all available sessions, run the following command:
+$ nox -l
+"""
+
+import os
+from pathlib import Path
+
+import nox
+import tomli
+from nox.sessions import Session
+
+# default sessions (sorted alphabetically)
+nox.options.sessions = ["bandit", "black", "flake8", "isort", "mypy", "unit-tests-python"]
```

Comment:
Do you agree with `unit-tests-python-3.x` or do you prefer a shorter name ?

Reply:
I would have shortened this to `utests` for the sake of those who would want to invoke just this section (less typing to do), but that's not binding.

Reply:
I'd say `.*python.*` is redundant in the context of unit tests

Reply:
this is pretty much the last thread that needs to be resolved

Reply:
I think this is nitpick territory. Let's resolve it and merge the PR.